com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/MotionBlur.compute

```diff
@@ -162,13 +175,53 @@ float4 ProcessSample(uint sampleNumber, float2 dir, float invSampleCount, float2
     return float4(sampleColor, sampleWeight);
 }
 
+float4 ProcessSampleFastPath(uint sampleNumber, float2 dir, float invSampleCount, float2 centralUV, float randomVal, float dirSign)
```

Comment:
ProcessSampleFastPathF4 is exactly same than ProcessSampleFastPath, don't think it is expect?

Reply:
My bad. you are right.